Sources/MongoSwift/AsyncAwait/MongoDatabase+AsyncAwait.swift

```diff
@@ -27,6 +27,8 @@ extension MongoDatabase {
      *
      * - Throws:
      *   - `MongoError.CommandError` if an error occurs that prevents the command from executing.
+     *     If a collection with this name already exists, a `MongoError.CommandError` with code `48`
```

Comment:
Unfortunately, due to SERVER-64416, this isn't actually always the case. Not sure if that's worth calling out here or not.